src/python/nimbusml/internal/core/preprocessing/schema/prefixcolumnconcatenator.py

```diff
@@ -15,19 +16,12 @@
 from ...base_pipeline_item import BasePipelineItem, DefaultSignature
 
 
-class PrefixColumnConcatenator(BasePipelineItem, DefaultSignature):
+class PrefixColumnConcatenator(
+        BasePipelineItem,
+        DefaultSignature):
     """
 
-    Combines several columns into a single vector-valued column by prefix
```

Comment:
>Combines several columns into a single vector-valued column by prefix [](start = 4, length = 69)

need this doc

Reply:
Fixed in latest commit.

---
In reply to: [346961442](https://github.com/microsoft/NimbusML/pull/364#discussion_r346961442) [](ancestors = 346961442)